clifford/test/test_degenerate.py

```diff
@@ -5,14 +5,18 @@
 import numpy as np
 
 
-class PGATests(unittest.TestCase):
-
-    def setUp(self):
-        self.algebras = [Cl(3, 0, 1, firstIdx=0), Cl(4, 0, 1, firstIdx=0)]
-
-    def test_integration(self):
-        layout = self.algebras[0][0]
-        blades = self.algebras[0][1]
+class TestPGA:
+    layout, _ = Cl(3, 0, 1, firstIdx=0)
+
+    def test_right_complement(self):
+        # TODO[gh-216]: Does this mean we implemented the left complement?
+        for b in self.layout.blades_list:
+            assert b.right_complement() * b == self.layout.pseudoScalar
```

Comment:
@enkimute: should this be (the passing):
```suggestion
            assert b.right_complement() ^ b == self.layout.pseudoScalar
```
or even the failing
```suggestion
            assert b ^ b.right_complement() == self.layout.pseudoScalar
```
?

Reply:
```
assert b ^ b.right_complement() == self.layout.pseudoScalar
assert b.left_complement() ^ b == self.layout.pseudoScalar
```